fe/fe-core/src/test/resources/sql/optimized-plan/lateral.sql

```diff
@@ -8,8 +8,7 @@ TABLE FUNCTION (unnest)
 [sql]
 select v1 from tarray, lateral unnest(v3)
```

Comment:
It seems these two plans are not equal. The old plan will expand rows in tarray, like v1 = 3, v3= [1,2,3] will expand to v1= 3, unnest = 1; v1= 3, unnest = 2; v1= 3, unnest = 3;